xbmc/storage/MediaManager.cpp

```diff
@@ -64,23 +64,21 @@ const char MEDIA_SOURCES_XML[] = { "special://profile/mediasources.xml" };
 CMediaManager::CMediaManager()
 {
   m_bhasoptical = false;
```

Comment:
eventually also default initialize `m_bhasoptical` and make the ctor = default. but thats up to you

Reply:
Nah I don't want to touch that now nor with this minor change. HasOptical clearly doesn't belong in the manager and is currently used only in Windows code path. I am still wrapping my head around a possible "design" for all this but each step at a time :)